### ISSUE TITLE
Windows 10 Support

### ISSUE THREAD

**Issue by Allmythings1066** [floatinghotpot/cordova-plugin-nativeaudio#49]
Are there any plans to support Windows 8.1 (incl. Win 10)?
Thanks

**Comment by floatinghotpot** [comment]
Google did not update their SDK, I am not sure whether win10 can be supported or not.

You can try and send me your feedback, then let's see what can be done or not.

**Comment by floatinghotpot** [comment]
Sorry, no such plan so far.

**Comment by Allmythings1066** [comment]
What a shame - Windows 10 seems like it could be a growing market.  Anyway, thanks for all your brilliant work.

**Comment by floatinghotpot** [comment]
I don't have such positive opinion on Windows 10, as M$ still charge for Visual Studio, while Xcode and Android Studio is totally free, which encourage more developers for their platform.

At least I see, among so many mobile Ad networks, only Google AdMob offers a WP8 version, other vendors only provide iOS & Android SDK.

**Comment by Allmythings1066** [comment]
Yeah I see what you ate saying. I use the free version of VS and I was thinking that the Microsoft app store can only get bigger with Windows 10. Anyway, thanks very much for looking into it for me


Sent from my Samsung device

-------- Original message --------

**Comment by nowrap** [comment]
Any news on this?
There is a sunset jira ticket for the cordova-plugin-media. Suggestion is to use web audio api:
https://issues.apache.org/jira/browse/CB-13095

There is a MS sample app using WinJS:
https://github.com/Microsoft/Windows-universal-samples/tree/master/Samples/BackgroundMediaPlayback

I am primaly asking for Windows 10 (desktop) UWP apps.
